src/main/java/com/hivemq/metrics/HiveMQMetrics.java

```diff
@@ -86,16 +86,16 @@
      *
      * @since 3.0
      */
-    public static final HiveMQMetric<Gauge<Number>> BYTES_READ_TOTAL =
-            HiveMQMetric.gaugeValue("com.hivemq.networking.bytes.read.total");
+    public static final HiveMQMetric<Counter> BYTES_READ_TOTAL =
```

Comment:
IIrc influxdb and prometheus can't handle such a type change. This may brake the metric when HiveMQ is updated.